go/client/logclient.go

```diff
@@ -105,6 +101,19 @@ type getEntryAndProofResponse struct {
 	AuditPath []string `json:"audit_path"` // the corresponding proof
 }
 
+type getProofByHashResponse struct {
+	LeafIndex    int      `json:"leaf_index"` // The 0-based index of the end entity corresponding to the "hash" parameter.
+	AuditPath    []string `json:"audit_path"` // An array of base64-encoded Merkle Tree nodes proving the inclusion of the chosen certificate.
```

Comment:
Does it work if you make this field a `[][]byte` instead (and strip out the base64 decoding at line 423)?